public/js/actions/types.js

```diff
@@ -88,7 +87,7 @@ type SourceAction =
                sourceText: SourceText }}
   | { type: "CLOSE_TAB", id: string };
 
-type UIAction = { type: constants.TOGGLE_FILE_SEARCH, searchOn: boolean };
+type UIAction = { type: "TOGGLE_FILE_SEARCH", searchOn: boolean };
```

Comment:
It doesn't need to use the constants. Once everything is flow typed the constants file can go away. Flow will error if you pass in a string that doesn't match any of these types.